#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_cmusstatusdisplay>.dpatch by Carlos Eduardo Sotelo Pinto (krlos) <krlos.aqp@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Updating non dpatch changes to dpatch

@DPATCH@

--- cmus-2.2.0.orig/cmus-status-display
+++ cmus-2.2.0/cmus-status-display
@@ -19,7 +19,7 @@
 output()
 {
 	# write status to /tmp/cmus-status (not very useful though)
-	echo "$*" >> /tmp/cmus-status 2>&1
+	echo "$*" >> ~/.cmus-status 2>&1
 
 	# WMI (http://wmi.modprobe.de/)
 	#wmiremote -t "$*" &> /dev/null
